editor/editor_themes.cpp

```diff
@@ -702,6 +702,19 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 	theme->set_color("icon_focus_color", "Button", icon_focus_color);
 	theme->set_color("icon_pressed_color", "Button", icon_pressed_color);
 
+	// Variation for Editor Log filter buttons
+	theme->set_type_variation("editor_log_filter_button", "Button");
+	// When pressed, don't tint the icons with the accent color, just leave them normal.
+	theme->set_color("icon_pressed_color", "editor_log_filter_button", Color::named("WHITE"));
+	// When unpressed, dim the icons.
+	theme->set_color("icon_normal_color", "editor_log_filter_button", Color::named("DIM_GRAY"));
```

Comment:
This being the sole named constant here (above color is redundant as I commented) is a bit weird. Maybe some existing color can be reused?

Reply:
I used `font_disabled_color`, which is similar.